.github/workflows/examples.yml

```diff
@@ -15,52 +15,27 @@ env:
 jobs:
   tf-validate:
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        terraform_version: ["${{vars.TF_VERSION_LATEST}}"]
     permissions: {}
     steps:
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
-        with:
-          fetch-depth: 0
-      - run: echo "GO_VERSION=$(cat .go-version)" >> "${GITHUB_ENV}"
       - uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32
         with:
           go-version-file: 'go.mod'
-      - name: go build
-        run: go build -o terraform-plugin-dir/terraform-provider-mongodbatlas-dev_v99.99.99_x5 .
-      - name: override plugin
```

Comment:
Can you clarify how the override of our provider is handled now?

Reply:
it uses the recommended way: https://developer.hashicorp.com/terraform/cli/config/config-file#development-overrides-for-provider-developers
that we also recommend from our guides: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/development-setup.md#building